CONTRIBUTING.md

```
@@ -0,0 +1,91 @@
+
+This project uses [Conventional Commits](https://www.conventionalcommits.org/)
+with a scope. Release versioning is driven by commit messages via
+[semantic-release](https://semantic-release.gitbook.io/).
+
+Format: `type(scope): description`
+
+Common types: `feat`, `fix`, `refactor`, `chore`, `docs`, `test`, `build`, `ci`.
+
+Examples:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)